tiledb/bioimg/converters/scale.py

```diff
@@ -35,58 +26,52 @@ def _scale_tile(
 
         scaled_tile.append(slice(start, stop))
 
-    output[tile] = resize(
-        base[tuple(scaled_tile)],
-        slice_shape,
-        preserve_range=True,
-        order=order,
-        anti_aliasing=True,
-    )
+    output[tile] = resize(base[tuple(scaled_tile)], slice_shape, **resize_kwargs)
 
 
 class Scaler(object):
     def __init__(
         self,
-        base: tiledb.array,
-        base_axes: Axes,
+        base_shape: Tuple[int, ...],
+        base_axes: str,
         scale_factors: Sequence[float],
         scale_axes: str,
-        mode: ScalerMode,
-        order: int,
+        chunked: bool = False,
+        progressive: bool = False,
+        order: int = 1,
     ):
-        self._mode = mode
+        self._chunked = chunked
         self._order = order
```

Comment:
Better save all the resize kwargs since they are referenced more than once: 
`self._resize_kwargs = dict(order=order, preserve_range=True, anti_aliasing=True)`